exercises/alice_russell/ex02_oo_basics/src/ShapeSorter.cpp

```diff
@@ -0,0 +1,51 @@
+#include "ShapeSorter.h"
+
+bool ShapeSorter::compare_area(Shape* &s1, Shape* &s2)
```

Comment:
This is also the type of method where documentation is crucial so that it can be described what compare means in this context, i.e. compare returns true iff the first shape has a greater area than the second.